Server.js

```diff
@@ -97,4 +97,16 @@ Server.prototype.listen = function (port, host, callback) {
 	})
 
 	return this
+}
+
+/**
+ * Sets the list of supported subprotocols
+ * @param {array} [protocols]
+ */
+Server.prototype.acceptSubprotocols = function (protocols) {
+	var that = this
+
+	this.acceptedSubprotocols = protocols;
```

Comment:
We probably should initialize this value in the constructor to an empty array, what do you think?